src/registry/contract.ts

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
Just abstracting away some things for interacting with our registry smart contract.